.gitignore

```diff
@@ -14,3 +14,5 @@ __pycache__/
 .env
 
 static/logs.log
+
+\.idea/
```

Comment:
this doesnt belong as part of this PR.

Reply:
So should I create a separate PR?

Reply:
I can do that in 3 hours

Reply:
> So should I create a separate PR?

Noo. Please amend your commit to add which issue your PR closed. See: http://api.coala.io/en/latest/Developers/Git_Basics.html#follow-up

During amend you can remove the changes you made in ``.gitignore``